packages/shared/src/urbit/channel.ts

```diff
@@ -357,6 +371,53 @@ export interface Perm {
   group: Flag;
 }
 
+interface Bot {
+  nickname?: string;
+  avatar?: string;
+  origin?: string;
+  agent: string;
+}
+
+interface PostMetadataSchemaV1 {
+  type: string;
+  bot?: Bot;
+  rendererOverride?: string;
```

Comment:
do we have an immediate use for this? i think it'll definitely be nice to have, but maybe we should defer until we need it

Reply:
I don't I just remember us talking about it, we can remove